docs/input/docs/ci-systems/github.md

```diff
@@ -0,0 +1,79 @@
+---
+Order: 60
+Title: GitHub
+Description: Running on gh-actions
+---
+
+## specifics
+
+* `Upload-Artifacts`-Task
+
+  Uploading artifacts is currently not supported in Cake.Recipe. Please use the actions/upload-artifacts GitHub Action
+
+## example config
+
+```yaml
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - "release/**"
+      - "hotfix/**"
+      - "feature/**"
+    tags-ignore:
+      - "*"
```

Comment:
In a general example, the `tags-ignore` configuration should not be used.

Reply:
I "minimalized" the examples and made them "comparable" (I.e. they all now do the same...)